lib/checks/canDistribute.js

```diff
@@ -0,0 +1,24 @@
+const Node = require('../node');
+
+function canDistribute(node) {
```

Comment:
something like `isBinomialSquared` would be much better yes! thanks Lili

`canDistribute` is a lot more cases than the one this function does and would be confusing

I think there's a name for this rule where there's the minus sign - ideally it'd be called that, cause otherwise it sounds like (x+2)^2 is handled here too 

having trouble thinking of what exactly to call it